include/userInputParameters.h

```diff
@@ -250,6 +250,7 @@ class userInputParameters
   double                    buffer_between_grains;
 
   bool         load_grain_structure;
+  bool         load_unstructured_grid;
```

Comment:
Why is this here?

Reply:
It will not do harm. It is there because I broken up my all changes to do PR for multiple nuclei with same OP. This will be needed when I do the PR for vtk file read. But you can remove if it is making a problem.